.gitignore

```diff
@@ -27,6 +27,9 @@ share/python-wheels/
 *.egg
 MANIFEST
 
+
+.DS_Store
```

Comment:
hmm should we remove this since it wasnt in the original gitignore?

Reply:
fixed now @blackgirlbytes